features/support/env.rb

```diff
@@ -1,5 +1,7 @@
 require 'open3'
 
 def current_ip
```

Comment:
A comment might be useful here - does this just parse the output of `ifconfig` to obtain the current IP address, or are there additional steps?

Reply:
I've added a comment - hopefully the rest should be self-explanatory